### PR TITLE
Fix remainder

### PR DESCRIPTION
**Pull request by shibatch**
This patch fixes a bug in remainder function.

### CODE REVIEW

**Comment by fpetrogalli** [comment]
Hi. Could you guard the fix with a test on the input value that was causing the failure?

**Comment by shibatch** [comment]
Done.

File: src/libm-tester/tester.c [hunk 1]
```diff
@@ -4348,6 +4348,7 @@ void do_test() {
     for(y = -1e+7;y < 1e+7 && success;y += 1.51e+5) {
       for(x = -1e+7;x < 1e+7 && success;x += 1.51e+5) checkAccuracy_f_f(mpfr_remainder, child_remainderf, y, x, 0.5);
     }
+    checkAccuracy_f_f(mpfr_remainder, child_remainderf, 11114942644092928.0, 224544296009728.0, 0.5);
```
Comment (fpetrogalli) [inline src/libm-tester/tester.c:4351]: I think you want to add a test also for the double precision case.

Comment (shibatch) [reply src/libm-tester/tester.c:4351]: Only single precision cases are found.

Comment (fpetrogalli) [reply src/libm-tester/tester.c:4351]: I see - if that's the case, I think you should revert the change in the double precision implementation.

Comment (shibatch) [reply src/libm-tester/tester.c:4351]: No. Those two implementations work with the same principle. So, if the single-precision version fails, there should be cases for the double-precision version to fail.

**Pull request merged**